app/views/registration_wizard/ineligible_for_funding/_not_in_england.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1 class="govuk-heading-xl">Funding eligibility</h1>
+
+<p class="govuk-body">
+  You're not elgibible for scholarship funding for <%= localise_sentence_embedded_course_name(course) %> because you do not work in England.
```

Comment:
```suggestion
  You’re not eligible for scholarship funding for <%= localise_sentence_embedded_course_name(course) %> because you do not work in England.
```